.github/workflows/ci.yml

```diff
@@ -15,20 +15,16 @@ on:
 
 jobs:
   test-build-full:
-    name: Test Linux, notebook + offscreen
+    name: Test Linux
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 10
```

Comment:
we can go back to 10 mins now that we don't have python3.10 which required more time due to no imgui wheels existing for it.